### PR TITLE
Add notice about autoloading AppCache via Composer

### PR DESCRIPTION
**Pull request by bocharsky-bw**
According to https://github.com/symfony/symfony-standard/pull/897 .

### CODE REVIEW

File: UPGRADE-2.8.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Standard Edition:
      overwrite the cache and log directories in your `AppKernel` as it is also done
      in Symfony 3 now (see
      [`app/AppKernel.php`](https://github.com/symfony/symfony-standard/blob/master/app/AppKernel.php#L31-L44)).
- * The `app/AppKernel.php` is now autoloaded via composer.
+ * The `app/AppKernel.php` and `app/AppCache.php` are now autoloaded via composer.
```
Comment (xabbuh) [inline UPGRADE-2.8.md]: […] files are now […]

Comment (bocharsky-bw) [inline UPGRADE-2.8.md]: Thanks!

**Comment by xabbuh** [comment]
:+1:

**Comment by Tobion** [comment]
Thank you @bocharsky-bw.

**Pull request merged**